config/checkstyle.xml

```diff
@@ -9,313 +9,304 @@ page at http://checkstyle.sourceforge.net/config.html -->
-            the same line. e.g., the following example is fine:
-            <pre>
-              if {
-                ...
-              } else
-            </pre>
-            -->
-            <!-- This next example is not fine:
-            <pre>
-              if {
-                ...
-              }
-              else
-            </pre>
-            -->
-            <property name="option" value="same"/>
-            <property name="severity" value="warning"/>
-        </module>
-
-        <!-- Checks for braces around if and else blocks -->
-        <module name="NeedBraces">
-            <property name="severity" value="warning"/>
-            <!--<property name="tokens" value="LITERAL_IF, LITERAL_ELSE, LITERAL_FOR, LITERAL_WHILE, LITERAL_DO"/>-->
-            <property name="tokens" value="LITERAL_ELSE, LITERAL_FOR, LITERAL_WHILE, LITERAL_DO"/>
-        </module>
-
-        <module name="UpperEll">
-            <!-- Checks that long constants are defined with an upper ell.-->
-            <property name="severity" value="error"/>
-        </module>
-
-        <module name="FallThrough">
-            <!-- Warn about falling through to the next case statement.  Similar to
-            javac -Xlint:fallthrough, but the check is suppressed if a single-line comment
-            on the last non-blank line preceding the fallen-into case contains 'fall through' (or
-            some other variants which we don't publicized to promote consistency).
-            -->
-            <property name="reliefPattern"
-                      value="fall through|Fall through|fallthru|Fallthru|falls through|Falls through|fallthrough|Fallthrough|No break|NO break|no break|continue on"/>
-            <property name="severity" value="error"/>
-        </module>
-
-
-        <!--
-
-        MODIFIERS CHECKS
 
-        -->
+    <!--
+
+    MODIFIERS CHECKS
+
+    -->
+
+    <module name="ModifierOrder">
+      <!-- Warn if modifier order is inconsistent with JLS3 8.1.1, 8.3.1, and
+             8.4.3.  The prescribed order is:
+             public, protected, private, abstract, static, final, transient, volatile,
+             synchronized, native, strictfp
+          -->
+    </module>
 
-        <module name="ModifierOrder">
-            <!-- Warn if modifier order is inconsistent with JLS3 8.1.1, 8.3.1, and
-                 8.4.3.  The prescribed order is:
-                 public, protected, private, abstract, static, final, transient, volatile,
-                 synchronized, native, strictfp
-              -->
-        </module>
 
+    <!--
 
-        <!--
+    WHITESPACE CHECKS
 
-        WHITESPACE CHECKS
+    -->
 
+    <module name="WhitespaceAround">
+      <!-- Checks that various tokens are surrounded by whitespace.
+             This includes most binary operators and keywords followed
+             by regular or curly braces.
         -->
+      <property name="tokens" value="ASSIGN, BAND, BAND_ASSIGN, BOR,
```

Comment:
The difference between this, and just accepting the default are the following values:
* `DO_WHILE` (the `while` keyword in a do-while)
* `LCURLY` (left curly) 
* `LITERAL_SWITCH` (the `switch` keyword)
* `RCURLY`(right curly)
* `SLIST` (a statement list)
* `LITERAL_ASSERT` (the `assert` keyword)
* `TYPE_EXTENSION_AND` (the `&` symbol when used in a generic upper or lower bounds constrain)

If there isn't a particular reason for excluding these values, this entire block can be simplified to `module name="WhitespaceAround"/>`